Assets/DeathBase.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public abstract class DeathBase : MonoBehaviour
+{
+    [SerializeField]
+    private Camera _mainCamera = null;
+
+    [SerializeField]
+    protected bool _isDead = false;
+    public bool IsDead => _isDead;
+    
+    private readonly Rect _viewRect = new Rect(0f, 0f, 1f, 1f);
```

Comment:
```suggestion
    [SerializeField]
    private Camera _mainCamera = null;
    
    private readonly Rect _viewRect = new Rect(0f, 0f, 1f, 1f);
    
    [SerializeField]
    protected bool _isDead = false;
    
    
    public bool IsDead => _isDead;

```

　クラス変数のスコープは　それぞれの属性にあわせて　まとめておきたいです